tests/cover/test_datetimes.py

```diff
@@ -143,19 +143,15 @@ def test_single_date(val):
 
 
 def test_can_find_midnight():
-    minimal(times(), lambda x: x.hour == x.minute == x.second == 0)
+    times().filter(lambda x: x.hour == x.minute == x.second == 0).example()
 
 
 def test_can_find_non_midnight():
     assert minimal(times(), lambda x: x.hour != 0).hour == 1
 
 
-def test_can_find_on_the_minute():
-    minimal(times(), lambda x: x.second == 0)
```

Comment:
Mildly surprised that `.second` and `.seconds` both exist as distinct attributes, but okay.

I’d lean towards keeping it anyway, (1) as a counterpart to `test_can_find_off_the_minute`, and (2) so we can distinguish between a failure where we can’t find midnight but can find on the minute for other minutes. But don’t feel strongly.